src/styles/value.scss

```diff
@@ -0,0 +1,13 @@
+$number-of-days: 31;
+$number-of-cells: 7;
+$cell-size: 100px;
+$calendar-width: $number-of-cells * $cell-size + ($number-of-cells - 1);
+$margin-day: $cell-size + 1px;
+$days:
+  "mon" 0,
+  "tue" 1 * $margin-day,
+  "wed" 2 * $margin-day,
+  "thu" 3 * $margin-day,
+  "fri" 4 * $margin-day,
+  "sat" 5 * $margin-day,
+  "sun" 6 * $margin-day;
```

Comment:
you could remove $margin-day from here and move it to your loop directly for avoid duplicates

Reply:
I have no idea how to do this, can you give me a hint?